lib/web3/event.js

```diff
@@ -138,17 +138,16 @@ SolidityEvent.prototype.encode = function (indexed, options) {
  * @return {Object} result object with decoded indexed && not indexed params
  */
 SolidityEvent.prototype.decode = function (data) {
- 
-    data.data = data.data || '';
-    data.topics = data.topics || [];
+    const dataData = data.data || '';
```

Comment:
I need help from someone who's more familiar with the code base than I am for this one.